package.json

```diff
@@ -89,7 +92,7 @@
     "build:clean": "rm -rf dist bower-dist app/generated .tmp",
     "build:mkdist": "mkdir dist; mkdir -p dist/$npm_package_name-$npm_package_version/styles; (cd dist; ln -s $npm_package_name-$npm_package_version latest)",
     "build:version": "node -e 'require(\"fs\").writeFileSync(\"app/version.json\", JSON.stringify({lformsVersion: require(\"./package.json\").version}))'",
-    "build:css:lforms": "cleancss --skip-rebase -o dist/latest/styles/lforms.min.css dist/latest/styles/lforms.css",
+    "build:css:lforms": "cleancss --skip-rebase -o dist/latest/styles/lforms.min.css dist/lforms-0.0.3/styles/lforms.css",
```

Comment:
@semicolin  that is correct, but i was face issue while publish lforms on npm  so i keep it form specified location.

Reply:
It looks like the `build:mkdist` script creates a symlink for `dist/latest`. is that working correctly? I don't understand why it would work for the cleancss output file but not the input file.

Reply:
Hi @semicolin Don't know how but the `$npm_package_name`  variable is referring to the correct package name, and that's why the issue occurs while creating directory to add css and other files. please check below ss.
It should create` test-0.0.5` dir and add all css the files in it as per the scripts. But strangely it is not working. Can you please check this at your side also ?
![Screenshot from 2023-02-07 20-04-37](https://user-images.githubusercontent.com/92708869/217275235-6d2a9ecf-b45f-4567-a3b1-0b477c04dee9.png)

Reply:
webpack.config.js line 55 has hardcoded package name. Change it to:
```js
let versionedDist = require('./package.json').name+'-'+require('./package.json').version;
```

Reply:
okay @semicolin   I will update